Library/ENV/4.3/cc

```diff
@@ -1,6 +1,10 @@
-#!/System/Library/Frameworks/Ruby.framework/Versions/Current/usr/bin/ruby -W0
+#!/bin/sh
+# Make sure this shim uses the same Ruby interpreter that is used by Homebrew.
+unset RUBYLIB
+unset RUBYOPT
+exec "$HOMEBREW_RUBY_PATH" -x "$0" "$@"
```

Comment:
I think let's keep it simple. i.e. make SCM wrapper shows same message as `cc`. If needed, we can reevaluate it if someone really sees such failure message, which is very unlikely.